### PR TITLE
[Feature] Doing RSA Asymmetrical Encryption to generate Session Id

### PR DESCRIPTION
**Pull request by sammous**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,14 +16,26 @@ To be able to use XenIssuing, you will need to use a private key provided by Xen
...
+- `generateSessionId` will encrypt a session key randomly generated used for asymmetric encryption with Xenissuing.
 - `encrypt` would be used when setting sensitive data.
 - `decrypt` would be used whenever receiving sensitive data from Xenissuing.
 
+
+### Session ID
+
+```swift
+let xen = try! Xenissuing(xenditPublicKeyData: Data(base64Encoded: "BASE64_PUBLIC_KEY")!)
+let sessionKey = try! xcrypt.generateRandom()
```
Comment (UzaiSindiko) [inline README.md]: is this suppose to be `xcrypt` or `xen`?

Comment (sammous) [reply README.md]: correct, typo thanks 🙏🏽

**Review by 0505gonzalez — CHANGES_REQUESTED** [review]
@sammous can we use this interface? https://xendit.slack.com/archives/C02CWFJQCM8/p1674023025339439

**Comment by 0505gonzalez** [comment]
Also, can we call the class `XenIssuing` instead of `XenCrypt`? Since `XenIssuing` is the official brand and in the future this SDK may do more than encryption

**Comment by sammous** [comment]
which class you mean ? The module `XenCrypt` would be where all encryption related topics are, else the main class is called `Xenissuing`, cc https://github.com/xendit/xenissuing-ios/tree/main/Sources/Xenissuing

**Pull request merged**